foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -161,14 +161,12 @@ Writing an HTML comment is simple: we just put `<!--` and `-->` at either end of
 <div class="lesson-content__panel" markdown="1">
 1. Watch Kevin Powell's [HTML Paragraph and Headings Video](https://www.youtube.com/watch?v=yqcd-XkxZNM&list=PL4-IK0AVhVjM0xE0K2uZRvsM7LkIhsPT-&index=3)
 2. Watch Kevin Powell's [HTML Bold and Italic Text Video](https://www.youtube.com/watch?v=gW6cBZLUk6M&list=PL4-IK0AVhVjM0xE0K2uZRvsM7LkIhsPT-&index=4)
-3. To get some practice working with text in HTML, create a plain blog article page which uses different headings, uses paragraphs, and has some text in the paragraphs bolded and italicized. You can use [Lorem Ipsum](https://loremipsum.io/) as the dummy text for the article content and headings.
+3. To get some practice working with text in HTML, create a plain blog article page which uses different headings, uses paragraphs, and has some text in the paragraphs bolded and italicized. You can use [Lorem Ipsum](https://loremipsum.io/) to generate dummy text, in place of real text as you build your sites.
 </div>
 
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-- Note: In the examples, you have probably noticed that we used some strange looking text. This is [Lorem Ipsum](https://loremipsum.io/). It is used to generate dummy text you can use in place of real text as you build your sites.
-
```

Comment:
Since the Additional Resources are now empty, can we add the following list item in that section:

*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.

Reply:
yeah, you're right, i realized it later on, thanks for bringing that up, much appricated....